src/main/scala/mesosphere/marathon/api/HTTPMetricsFilter.scala

```diff
@@ -3,15 +3,18 @@ package api
 
 import javax.servlet._
 import javax.servlet.http.{HttpServletRequest, HttpServletRequestWrapper, HttpServletResponse, HttpServletResponseWrapper}
-import mesosphere.marathon.metrics.{Metrics, ServiceMetric}
+import kamon.metric.instrument.Memory
+import mesosphere.marathon.metrics.Metrics
 
 /**
   * This filter replaces the default I/O streams with proxies that count
   * the number of bytes that are sent to and from the client.
   */
 class HTTPMetricsFilter extends Filter {
-  private[this] val bytesReadMetric = Metrics.counter(ServiceMetric, getClass, "bytesRead")
-  private[this] val bytesWrittenMetric = Metrics.counter(ServiceMetric, getClass, "bytesWritten")
+  private[this] val bytesReadMetric =
+    Metrics.counter("marathon.http.data.read.bytes.total", unit = Memory.Bytes)
```

Comment:
I don't really like repeating that much of information on every instrument instantiation. Mainly because this is a recipe for trouble. That's because someone can easily bypass the rules we have right now in mind and come up with a totally ridiculous naming in the future.